kep/119-node-resource-topology-aware-scheduling/README.md

```diff
@@ -0,0 +1,261 @@
+```
+
+`serviceAccountName: noderesourcetopology-account` would have to be added to the manifest file of the scheduler deployment file.
+
+# Use cases
+
+Numbers of kubernetes worker nodes on bare metal with NUMA topology. TopologyManager feature gate enabled on the nodes. In this configuration, the operator does not want that in the case of an unsatisfactory host topology, it should be re-scheduled for launch, but wants the scheduling to be successful the first time.
+
+# Known limitations
+
+Kube-scheduler makes an assumption about current resource usage on the worker node, since kube-scheduler knows which pod assigned to node. This assumption makes right after kube-scheduler choose a node. But in case of scheduling with NUMA topology only TopologyManager on the worker node knows exact NUMA node used by pod, this information about NUMA node delivers to kube-scheduler with latency. In this case kube-scheduler will not know actual NUMA topology until topology exporter will send it back. It could be mitigated if kube-scheduler in proposed plugin will add a hint on which NUMA id pod could be assigned, further Topology Manager on the worker node may take it into account.
+
+# Test plans
+
+It would be ensured that the components developed or modified for this feature can be easily tested.
+
+* Unit Tests
+
+Unit test for scheduler plugin (pkg/noderesources/node_resource_topology_match.go)
+pkg/noderesources/node_resource_topology_match_test.go which test the plugin.
+
+Separate tests for CRD informer also should be implemented.
+
+* Integration Tests
+   *  Default configuration (this plugin is disabled)
+     * no side effect on basic scheduling flow (and performance)
+     * no side effect no matter the CRD is installed or not
+
+   *  Enable this plugin
+     * basic workflow of this feature works (decision by scheduler is admitted by kubelet)
+     * basic negative path of this feature works (decision by scheduler is rejected by kubelet)
+     * verify the behavior when the CRD is and isn't installed
+
+* End-to-end tests
+
+Integration and End-to-end would Implementation of it does not constitute a difficulty, but requires appropriate multi-numa hardware for comprehensive testing of this feature. Comprehensive E2E testing of this would be done in order to graduate this feature from Alpha to Beta.
+
+# Graduation criteria
+
+* Alpha (v1.21)
+
+Following changes are required:
+- [ ] New `kube scheduler plugin` NodeResourceTopologyMatch.
+    - [ ] Implementation of Filter
+- [ ] Unit tests and integration tests from [Test plans](#test-plans).
+
+* Beta
+- [ ] Add node E2E tests.
+- [ ] Provide beta-level documentation.
+
+# Production Readiness Review Questionnaire
+
+## Feature enablement and rollback
+* **How can this feature be enabled / disabled in a live cluster?**
+    - This plugin doesn't require special feature gate, but it expects: TopologyManager and CPUManager feature gate enabled on the worker node\
```

Comment:
If I'm not wrong if special  KubeSchedulerConfiguration is not defined, than plugin is not loading.